internal/schema/fabric_common_mapping_helpers.go

```diff
@@ -0,0 +1,26 @@
+package schema
+
+import (
+	v4 "github.com/equinix-labs/fabric-go/fabric/v4"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func OrderGoToTerra(order *v4.Order) *schema.Set {
```

Comment:
This method should already be in `internal/fabric/schema/fabric_common_mapping_helpers.go`